test/test-server/ccloud_handlers.go

```diff
@@ -377,6 +377,9 @@ func handleLaunchDarkly(t *testing.T) http.HandlerFunc {
 			"cli.deprecation_notices":                []map[string]any{},
 			"cli.client_quotas.enable":               true,
 			"cli.stream_designer.source_code.enable": true,
+			"flink.rbac.namespace.cli.enable":        true,
+			"auth.rbac.identity_admin.enable":        false,
```

Comment:
Should we make this true so we can test it?

Reply:
I just copied the value from `cpd`, since that is where we were originally fetching the flag from.

Reply:
got it... we should probably still make it true so we can have the proper test coverage. if it breaks tests, let's make a separate pr where we make this true and fix the tests